src/waveform-builder.js

```diff
@@ -96,6 +108,9 @@
     if (options.dataUri) {
       return this._getRemoteWaveformData(options, callback);
     }
+    else if (options.rawData) {
```

Comment:
We should add some test cases (test/api-spec.js). And (not essential) but we could consider adding or changing one of the demo pages to show how to use this feature.

Reply:
Sure, I can try adding some tests using the test data.

Reply:
Thanks!

Reply:
tests created!